wordpress-mobile/gutenberg-mobile
# Move AppContainer and MainScreen to @wordpress/edit-post as Editor and Layout

Step 8b of https://github.com/wordpress-mobile/gutenberg-mobile/issues/958
This is a refactor PR, the goal is to move our remaining components to gutenberg.

Gutenberg PR https://github.com/WordPress/gutenberg/pull/16260

### Changes
- `App` (`src/app/App.js`) -> included in `RootComponent` in `src/index.js`
- `AppContainer` (`src/app/AppContainer.js`) -> renamed to `Editor` and moved to `gutenberg/packages/edit-post/src/editor.native.js`
- `MainScreen` (`src/app/MainScreen.js`) -> renamed to `Layout` and moved to `gutenberg/packages/edit-post/src/components/layout/index.native.js`
- `src/app/App.test.js` -> moved to `src/index.test.js` (cannot be moved to gutenberg at this point as it uses the initial-html as input)
- `src/app/AppContainer.test.js` -> moved to `gutenberg/packages/edit-post/src/test/editor.native.js`
- `src/app/style.scss` -> moved to `gutenberg/packages/edit-post/src/components/layout/style.native.js`
- `src/app/initial-html.js` -> moved to `src/initial-html.js` 

### Testing Instructions
- Boot the app on iOS and Android
- Check that the app works as expected.
- Test on WPAndroid and WPiOS
- Check that the app can be translated
- Check that the formatting works
- Check everything ;)

Update release notes:
